.travis.yml

```diff
@@ -12,6 +12,7 @@ sudo: false
 before_install:
   - pip install autopep8 hacking
   - pip install pytest-cov coveralls
+  - pip install Pillow
```

Comment:
Won't you write in lower case?

Reply:
it comes from official guide https://pillow.readthedocs.io/en/5.0.0/installation.html#basic-installation